visidata/column.py

```diff
@@ -179,6 +179,13 @@ def _calcIntoCache(self, row):
     def getValue(self, row):
         'Memoize calcValue with key sheet.rowid(row)'
 
+        if getattr(self.sheet, 'defer', False):
```

Comment:
I decided to have the defer=True case first because it was so much shorter than the defer=False case.